Tasks/QuickPerfTest/Invoke-QuickPerfTest.ps1

```diff
@@ -226,6 +240,12 @@ $CltAccountUrl = ComposeAccountUrl($VSOAccountUrl)
 
 Write-Verbose "VSO account Url = $VSOAccountUrl" -Verbose
 
+$tfsUrl = $env:System_TeamFoundationCollectionUri.TrimEnd('/')
+
+$resultsMDFolder = New-Item -ItemType Directory -Force -Path "$env:Temp\LoadTestResultSummary"
+Remove-Item $resultsMDFolder\QuickPerfTestResults_*.md -Force
+$summaryFile =  ("{0}\QuickPerfTestResults_{1}_{2}.md" -f $resultsMDFolder, $env:SYSTEM_DEFINITIONID, $env:BUILD_BUILDID)
```

Comment:
added run id instead

---
In reply to: [73854138](https://github.com/Microsoft/vsts-tasks/pull/2238#discussion_r73854138) [](ancestors = 73854138)